src/hooks/core/websocket/types.d.ts

```diff
@@ -111,6 +113,7 @@ export type GetstreamWebsocket = {
   unread_channels: number;
   unread_count: number;
   targetName: string;
+  reply_data?: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to line 30, the type of `reply_data` is marked as `any`. Consider replacing `any` with a more specific type.
```diff
-  reply_data?: any;
+  reply_data?: ReplyDataType; // Replace ReplyDataType with the actual type you expect.
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->